templates/api-template-nodejs/components/logger.js

```diff
@@ -87,18 +87,15 @@ module.exports = function () {
         return null;
     };
 
-    const log = (level, message) => {
-        const timestamp = new Date().toISOString();
-
+    const log = (level, message) => {        
         const logLevelMessageTypes = {
-            'error': `${timestamp}, 'ERROR \t', ${config.requestDetails}, ${message}`,
-            'warn': `${timestamp}, 'WARN \t', ${config.requestDetails}, ${message}`,
-            'info': `${timestamp}, 'INFO \t', ${config.requestDetails}, ${message}`,
-            'verbose': `${timestamp}, 'VERBOSE \t', ${config.requestDetails}, ${message}`,
-            'debug': `${timestamp}, 'DEBUG \t', ${config.requestDetails}, ${message}`,
-            'log': `${timestamp}, ${level} \t', ${config.requestDetails}, ${message}`
+            'error': message,
```

Comment:
sorry, you need to update in all the templates and remove the comas after reverting the change

Reply:
Is this not going to break the `log level` parsing logic in the log streamers?

Reply:
testing for node template. once it showing in correct format then will replicate the same on all 6 templates.

Reply:
Updated as format of log streamers.